static/app/components/draggableTabs/draggableTabList.tsx

```diff
@@ -132,6 +133,7 @@ function BaseDraggableTabList({
                 }
                 ref={element => (tabItemsRef.current[item.key] = element)}
               />
+
               {state.selectedKey !== item.key &&
                 state.collection.getKeyAfter(item.key) !== state.selectedKey && (
                   <TabDivider />
```

Comment:
This part of the DraggableTabListComponent is the only part that is sort of  specific to this implementation (or more specifically, the "filled" tab variant).